.ci/azure-pipelines/build-ubuntu-19-10.yaml

```diff
@@ -33,12 +34,6 @@ jobs:
         displayName: 'CMake Configuration'
       - script: |
           cd $BUILD_DIR
-          # Compiling some of the test targets with -j2 option leads to pipeline failures
-          # (presumably out of memory error). Thus we make them separately in a single
-          # thread mode. Their corresponding modules are built before with the -j2 mode
-          # to make the process faster.
-          cmake --build . -- -j2 pcl_filters pcl_registration
-          cmake --build . -- test_filters test_registration test_registration_api
```

Comment:
These commands are to prevent out of memory (RAM) issues, your PR is to address out of HDD space. There's no reason to touch these at the moment.

Reply:
Removed it from the PR. I was testing it out and wasn't getting any memory issues. Will test out more and if necessary create a new issue.

Reply:
Oh ok. Well, test away then. :)